### DISCUSSION TITLE
"Save" Button In Addition to "OK" Button

### DISCUSSION THREAD

**Discussion by Danc2050** [psoukie/zipchord#109] [Ideas]
Would it be a good feature to add a "Save" button that saves the changes, but does not close the GUI? I found that when I wanted to test out ZipChorder, I wanted the GUI up while I had a notepad in another part of my screen to see what options I liked or wanted to tweak.

Here is an example:
![image](https://user-images.githubusercontent.com/32185616/211486425-35a21434-a3d9-4ffd-bfef-bfeec1271d27.png)

If you think it is a good feature, I could also submit a PR for review as I have code as well since I used ChatGPT to give me direct questions to all my AHK questions :D.

**Comment by psoukie** [comment]
Great suggestion. I can look at it tonight after work. Feel free to open PR.

I'd prefer if the button said "Apply" and be to the left of "OK."

My approach would be to turn the ButtonOK function into ApplyMainSettings (and remove the CloseMainDialog call at the end). And the new ButtonOK would just call them (Apply... and Close...)  But happy to review what you came up with!
